File: docs/src/testers/guides/wait_time.ts
// @sniptest filename=wait_time.ts
// @sniptest show=1-10
import { NotteClient } from 'notte-sdk';

const client = new NotteClient();

const session = client.Session();
await session.use(async () => {

  // Wait 2 seconds
  await session.execute({ type: "wait", time_ms: 2000 });
});

const status = await session.status();

export { status };

File: docs/src/testers/guides/wait_time.py
# @sniptest filename=wait_time.py
# @sniptest show=1-7
from notte_sdk import NotteClient

client = NotteClient()

with client.Session() as session:
    # Wait 2 seconds before continuing
    session.execute(type="wait", time_ms=2000)

status = session.status()
